### PR TITLE
Document Cloud Controller Audit events

### PR DESCRIPTION
**Pull request by ewrenn8**
Please don't merge until @zrob approves.

Thanks!
Eli && Chris

### CODE REVIEW

File: managing-cf/audit-events.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+
+### Differences from other types of events
+Despite being similar in name, audit events are different from [Cloud Foundry Usage Events](./usage-events.html).
+Unlike usage events, audit events are **not** suitable for application and service billing.
+
+Another tool to help administrators monitor user actions on Cloud Foundry is [Security Event Logging](../../loggregator/cc-uaa-logging.html).
+Unlike Security Event Logs, audit events do **not** cover all API endpoints, but provide more details about a user's actions.
+
+
+### When are events created?
```
Comment (zrob) [inline managing-cf/audit-events.html.md.erb]: Let's re-order some of this to try and get the meat near the top

Overview
- differences
Using
- all the stuff in it
Event creation
Considerations

File: managing-cf/audit-events.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+
+Crash events (`audit.app.crash`) are created when Diego marks an application instance as crashed.
+
+Blob events (`blob.remove_orphan`) are created by a daily clean up job and do not represent user action.
+
+### Considerations
+Audit events can be created at any point during the execution of the action they describe. This means the action associated with the event is **not** guaranteed
+to have succeeded.
+
+DB migrations may result in the truncation of the audit events table, as the table tends to get quite large.  This means that a deployment with such a migration
```
Comment (zrob) [inline managing-cf/audit-events.html.md.erb]: reword this to make it clear that this is not common and it will be detailed in release notes

File: managing-cf/audit-events.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+### Querying For Events
+
+A user is able to query the events endpoint with various filters.  For example, a user could query for all events of a given type, or all events associated with a
+resource since a given timestamp.
+Allowed filters are enumerated on the [API documentation](https://apidocs.cloudfoundry.org/latest-release/events/list_all_events.html).
+
+### External Auditing Warehouse
+
+For auditing and compliance reasons, some operators may wish to keep a record of all audit events that have occured. Because audit events expire
+and may be purged during a deploy, these operators are must maintain an external data warehouse. This way they can store and access events across expiration events
```
Comment (zrob) [inline managing-cf/audit-events.html.md.erb]: "are must"

Comment (tcdowney) [reply managing-cf/audit-events.html.md.erb]: https://github.com/cloudfoundry/docs-running-cf/pull/60/commits/04694449662d6e0cbf66445e09770204cb416424#diff-ba05cd371854e68a073991abb3da83e4L92

File: managing-cf/audit-events.html.md.erb (hunk 2 above)
Comment (tcdowney) [reply managing-cf/audit-events.html.md.erb]: https://github.com/cloudfoundry/docs-running-cf/pull/60/commits/04694449662d6e0cbf66445e09770204cb416424#diff-ba05cd371854e68a073991abb3da83e4R206

File: managing-cf/audit-events.html.md.erb (hunk 1 above)
Comment (tcdowney) [reply managing-cf/audit-events.html.md.erb]: I left the "Viewing Audit Events" part in the Overview section and moved the rest as suggested.
https://github.com/cloudfoundry/docs-running-cf/pull/60/commits/04694449662d6e0cbf66445e09770204cb416424#diff-ba05cd371854e68a073991abb3da83e4R190

**Comment by zrob** [comment]
this looks good  👍
